kobo/hub/models.py

```diff
@@ -256,7 +256,11 @@ def update_worker(self, enabled, ready, task_count):
         Update only if data provided from a worker differs.
         """
         if (self.enabled, self.ready, self.task_count) != (enabled, ready, task_count):
+            self.enabled = enabled
```

Comment:
Ha... as perverse as it seems, "fixing" this might actually break something.  If the method has never worked since it was created, then any existing callers might be relying on the fact that it doesn't do anything.

I'm in favor of fixing it but then I must take some time to review callers of this method in repos using kobo.

Reply:
I also think this could break something else, I'll look around or leave the "broken" code.